pkg/pipeline/templates/terraformed.go.tmpl

```diff
@@ -79,8 +79,7 @@ func (tr *{{ .CRD.Kind }}) LateInitialize(attrs []byte) (bool, error) {
 	if err := json.TFParser.Unmarshal(attrs, params); err != nil {
 		return false, errors.Wrap(err, "failed to unmarshal Terraform state parameters for late-initialization")
 	}
-	opts := []resource.GenericLateInitializerOption{resource.WithZeroValueJSONOmitEmptyFilter(resource.CNameWildcard),
-	    resource.WithZeroElemPtrFilter(resource.CNameWildcard)}
+	opts := []resource.GenericLateInitializerOption{resource.WithZeroValueJSONOmitEmptyFilter(resource.CNameWildcard)}
```

Comment:
Why don't we need `resource.WithZeroElemPtrFilter(resource.CNameWildcard)` anymore?